pnpm-workspace.yaml

```diff
@@ -1,2 +1,5 @@
 packages:
-  - './packages/*'
\ No newline at end of file
+  - './packages/*'
+
+autoInstallPeers: false
+linkWorkspacePackages: true
```

Comment:
These options revert to their values that used to be the default in pnpm 7. In both cases, I genuinely think we're better off with them being configured this way.

Reply:
I've since set `linkWorkspacePackages` to `deep`, which I believe goes further than the pnpm 7 default, but is very desirable since packages in this repo have dependencies on their siblings that go multiple layers deep.